lib/parser.ts

```diff
@@ -55,6 +79,14 @@ function getUrlStringForChildrenLine(line: string, issue: Pick<GithubIssueData,
     const { owner, repo } = paramsFromUrl(issue.html_url)
     line = `${owner}/${repo}${line}`
   }
+  if (line.includes('/pull/')) {
+    /**
+     * getValidUrlFromInput requires a valid issue url, so we replace /pull/
+     * with /issues/ since github will recognize the pull request URL as a valid
+     * issue url
+     */
+    line = line.replace('/pull/', '/issues/')
+  }
```

Comment:
Pull requests are frequently used as children in task lists. Though not a primary use-case, we should support it.